CMakeLists.txt

```diff
@@ -467,6 +471,10 @@ ENDIF()
 ADD_EXECUTABLE(${TARGET} ${TARGET_SRC})
 DEAL_II_SETUP_TARGET(${TARGET})
 
+IF (ASPECT_PRECOMPILE_HEADERS AND CMAKE_VERSION VERSION_GREATER 3.15)
+  TARGET_PRECOMPILE_HEADERS(aspect PRIVATE <aspect/global.h> <aspect/introspection.h> <aspect/parameters.h> <deal.II/particles/particle_handler.h> <deal.II/base/table_handler.h> <deal.II/base/timer.h> <deal.II/base/conditional_ostream.h> <deal.II/distributed/tria.h> <deal.II/dofs/dof_handler.h> <deal.II/fe/fe.h> <deal.II/fe/mapping_q.h>)
```

Comment:
would you mind breaking this into one file per line? Maybe add a comment with the reasoning what to include and what to leave out?